### ISSUE TITLE
Differentiate browser / server errors by name

### ISSUE THREAD

**Issue by felixmosh** [MasterKale/SimpleWebAuthn#190]
Hi, since v5, looks like the `error.name` is always `WebAuthnError`, so if I want to show a translated message on my app, the only way to distinguish between errors is to check the message (which may change).

Previously this code (which is taken from the examples)
```js
try {
...
} catch(error) {
  if (error.name === 'InvalidStateError') {
    throw ['WEB_AUTHN.ERROR.DEVICE_ALREADY_EXIST'];
  } else if (error.name === 'NotAllowedError') {
    throw ['WEB_AUTHN.ERROR.NOT_ALLOWED'];
  }
}
```

Do you think that it is better to change the name of the error?
I can prepare a quick PR for it.

**Comment by felixmosh** [comment]
I've updated tests :]
thank you

**Comment by MasterKale** [comment]
The fix for this is now available in **@simplewebauthn/browser@5.1.0** 🚀
